JsonSchema.Data/UriIdentifier.cs

```diff
@@ -105,4 +110,20 @@ public override string ToString()
 	{
 		return Target.ToString();
 	}
+}
+
+[JsonSerializable(typeof(JsonSchema))]
+internal partial class DataGenerationSerializerContext : JsonSerializerContext
```

Comment:
@jevansaks why is this here?  copy/paste error?  There's one in the vocabularies file for this project already.

Reply:
Yeah, I admit I lost track sometimes of where I'd added JsonSerializerContexts. If there's another, let's switch to it.

Reply:
Taken care of in #638